lib/capistrano/tasks/database.rake

```diff
@@ -0,0 +1,109 @@
+namespace :forkcms do
+  namespace :database do
+
+    backup_file = 'mysql_backup.sql'
+
+    desc 'Create a backup of the database'
+    task :backup do
+      on roles(:web) do
+        # Fetch our flags to reach the database
+        flags = get_mysql_connection_flags
+
+        # Abort if no flags are found
+        if flags == nil
+          next
+        end
+
+        execute :mysqldump, "#{flags} --skip-lock-tables > #{release_path}/#{backup_file}"
```

Comment:
What happens if the file already exists?

Reply:
@tijsverkoyen it gets overwritten